Libplanet/Store/LiteDBStore.cs

```diff
@@ -48,14 +48,16 @@ public class LiteDBStore : BaseStore, IDisposable
         /// <param name="cacheSize">Max number of pages in the cache.</param>
         /// <param name="flush">Writes data direct to disk avoiding OS cache.  Turned on by default.
         /// </param>
-        /// <param name="readOnly">Opens database readonly mode. Turned off by default.
+        /// <param name="readOnly">Opens database readonly mode. Turned off by default.</param>
+        /// <param name="memory">Creates database in memory using <see cref="MemoryStream"/>.
         /// </param>
         public LiteDBStore(
             string path,
```

Comment:
Is this parameter ignored when `memory: true`?  If two parameters are mutually exclusive how about representing memory mode by passing `null` to the `path` parameter?